rero_ils/modules/notifications/tasks.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of RERO ILS.
+# Copyright (C) 2017 RERO.
+#
+# RERO ILS is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# RERO ILS is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with RERO ILS; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, RERO does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Celery tasks for notification records."""
+
+from __future__ import absolute_import, print_function
+
+from celery import shared_task
+from flask import current_app
+
+from ..loans.api import get_due_soon_loans, get_overdue_loans
+
+
+@shared_task(ignore_result=True)
+def create_over_and_due_soon_notifications():
+    """Creates due_soon and overdue notifications."""
+    over_due_loans = get_overdue_loans()
+    if over_due_loans:
```

Comment:
not required this can be removed get_overdue_loans shoud retrun always an array.